src/modules/ecommerceTracking/verifyOption.js

```diff
@@ -0,0 +1,17 @@
+/*
+param: string
+return: string
+*/
+
+export let verifyOption = (step) => {
+    let ecommerceOption;
+
+    //Checks the step of the ecommerce user journey and sets the ecommerce option accordingly
+    if(step === 'Step 4'){
```

Comment:
This could be turned into a conditional (ternary) operator to make it shorter.

Reply:
Done